src/utils.js

```diff
@@ -142,18 +142,28 @@ function debounce(fn, time) {
  * This is intended to be used to compose event handlers.
  * They are executed in order until one of them sets
  * `event.preventDownshiftDefault = true`.
- * @param {Function} fns the event handler functions
+ * @param {...Function} fns the event handler functions
  * @return {Function} the event handler to add to an element
  */
-function composeEventHandlers(...fns) {
+function callAllEventHandlers(...fns) {
   return (event, ...args) =>
     fns.some(fn => {
       fn && fn(event, ...args)
-      // TODO: remove everything after the || in the next breaking change
-      return event.preventDownshiftDefault || event.defaultPrevented
+      return event.preventDownshiftDefault
     })
 }
 
+/**
+ * This return a function that will call all the given functions with
+ * the arguments with which it's called. It does a null-check before
+ * attempting to call the functions and can take any number of functions.
+ * @param {...Function} fns the functions to call
+ * @return {Function} the function that calls all the functions
+ */
+function callAll(...fns) {
+  return (...args) => fns.forEach(fn => fn && fn(...args))
```

Comment:
from types point of view this would make sense to be
```js
function callAll(...fns) {
  return (...args) => {
    fns.forEach(fn => fn && fn(...args))
  }
}
```
u dont intend to return anything from returned function after all